BrowserStack.java

```diff
@@ -37,32 +38,113 @@ public static void myassert(boolean  x) {
     //push() - pushes the element in this case its a url (String)
     private  void push(String url){
         //WRITE YOUR CODE HERE
+    	
```

Comment:
Stack was initialized to size of 16 (STACK_SIZE). So, we should check for Stack full condition and return appropriate error.